### ISSUE TITLE
parse std::filesystem::copy as std::copy from <algorithm>

### ISSUE THREAD

**Issue by CareF** [cpplint/cpplint#200]
If there is a call of `std::filesystem::copy`, it will be falsely recognized as std::copy and trigger a false negative error report.

**Comment by tkruse** [comment]
Thanks for the report, can you provide a bit more details, like a few line of context, and the error message?
I am not sure if this can be easily fixed, but I wont have the time myself to do so.

**Comment by jiawen** [comment]
I too have this issue, along with related false positive in `<algorithm>`.

Examples:

```
namespace nda {

void copy(MyType src, MyType dst);

class MyClass {
public:
  void forward(MySecondType src);
};

}  // namespace nda

Running `cpplint` triggers:

```
file.cpp:<line>: Add #include <algorithm> for copy  [build/include_what_you_use] [4]
file.cpp<line>: Add #include <utility> for forward  [build/include_what_you_use] [4]
```

I'm an Xooger - I think this was less of an issue at Google because their style guide requires that functions be Capitalized.
